FORMAT_ADIOS.md

```diff
@@ -0,0 +1,30 @@
+Implementation Details: ADIOS1/2
+================================
+
+Datatype Conventions
+--------------------
+
+## Boolean Types
+
+ADIOS1 (BP3) was traditionally a C library and as such as no notion for boolean types.
+[At the moment](https://github.com/ornladios/ADIOS2/issues/2606), also ADIOS2 (BP4) does not define boolean types.
+Although the openPMD-standard avoids using boolean types itself, developers might choose to implement them as attributes or dataset types.
+
+For compatibility, boolean types shall be stored as `adios_unsigned_byte`/`unsigned char`.
+
+### ADIOS1
+
+ADIOS1 is deprecated.
+A stored boolean type is identified as `adios_unsigned_byte` on the read side, since no convention for bool-identification is specified.
+
+### ADIOS2
+
+For each boolean attribute or variable, an additionally attribute is stored adding identification for bool for data readers.
+These identifying attributes are prefixed with `__is_boolean__/` (for attributes) or `__is_boolean_var__` (for variables), and then adding the full path of the described attribute or variable.
```

Comment:
@franzpoeschel I am already defining the identifier for ADIOS2 boolean variables here as well: https://github.com/openPMD/openPMD-api/issues/921